crates/sui-indexer/migrations_v2/2023-08-19-044026_transactions/up.sql

```diff
@@ -17,11 +17,12 @@ CREATE TABLE transactions (
     transaction_kind            smallint     NOT NULL,
     -- number of successful commands in this transaction, bound by number of command
     -- in a programmaable transaction.
-    success_command_count       smallint     NOT NULL
-);
+    success_command_count       smallint     NOT NULL,
+    CONSTRAINT transactions_pkey PRIMARY KEY (tx_sequence_number, checkpoint_sequence_number)
+) PARTITION BY RANGE (checkpoint_sequence_number);
 
+CREATE TABLE transactions_partition_0 PARTITION OF transactions FOR VALUES FROM (0) TO (MAXVALUE);
```

Comment:
Curious. How do we populate this partition? There's only genesis in epoch 0 I think.

Reply:
this is empty on genesis, b/c it has range from 0 to maxvalue, so when we populate `transactions` on epoch 0, we actually populate into this partition

Reply:
Got it. I meant we only have genesis transaction in epoch 0